api_app/urls.py

```diff
@@ -13,14 +17,33 @@
 
 from .auth import LoginView, LogoutView
 
-
 # Routers provide an easy way of automatically determining the URL conf.
 router = routers.DefaultRouter(trailing_slash=False)
 router.register(r"tags", TagViewSet)
 router.register(r"jobs", JobViewSet)
 
+# Documentation Settings
+schema_view = get_schema_view(
```

Comment:
instead of adding this here, we should add a new file called `schemas.py` and add all this docs related code there.